app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -823,6 +823,22 @@ def test_allocate_buckets_deletes_buckets_preserves_occupied_isolation_group(sel
         self.assertEqual(NimbusBucketRange.objects.count(), 2)
         self.assertEqual(NimbusIsolationGroup.objects.count(), 1)
 
+    def test_bucket_namespace_changes_for_rollout(self):
+        feature = NimbusFeatureConfigFactory(slug="feature")
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            application=NimbusExperiment.Application.DESKTOP,
+            channel=NimbusExperiment.Channel.RELEASE,
+            feature_config=feature,
+            population_percent=Decimal("50.0"),
+        )
+        original_namespace = experiment.bucket_namespace
+
+        experiment.is_rollout = True
+        experiment.save()
+
+        self.assertNotEqual(original_namespace, experiment.bucket_namespace)
```

Comment:
Oh instead of doing this I would just do it like above in 

https://github.com/mozilla/experimenter/pull/6657/files#diff-b7f8cfcb629fab7e38ae45d46aa600746219a2fe6e73f6e8b98e3a95f21da0ebR748-R751

and explicitly assert the actual content of the namespace that includes the `rollout` key